Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUAdapterImpl.cpp

```diff
@@ -50,6 +50,11 @@ static Ref<SupportedFeatures> supportedFeatures(WGPUAdapter adapter)
 
     Vector<String> result;
     for (auto feature : features) {
+        if (feature == WGPUFeatureName_BGRA8UnormStorage) {
```

Comment:
Why are we doing this outside of the switch? Let's use casts instead.

Reply:
like this?
```
switch (static_cast<uint32_t>(feature))
```

we lose the benefit of the compiler catching we have not missed one of the values of the enum, but I am fine with that

Reply:
Now that WebGPU.h is updated, these changes are no longer needed :)